surveys/2021-annual-survey/design.md

```diff
@@ -39,29 +44,50 @@ While demographic information can be interesting on its own, it's even more inte
 * Which resources helped learners the most?
 
 ## Improving the language
+
 * Which language would you most likely want to interop with Rust?
+* What are people's frustrations with language, std.
+* What problems are you hitting? What problems haven't been solved yet?
+
+## Stability
 
-## Language Stability
 * How often does updating a compiler break people's code? How easy are breaking changes to deal with?
     * Care should be taken here to address how this changes if the user is using nightly vs stable.
 * How stable are people's dependencies?
     * We should not ask this in terms of number of 1.0 dependencies but rather how people perceive the stability of the dependencies they use.
     
 ## Using Rust
+
 * How long have people been using Rust?
     * We will want to qualify this perhaps ("as the main or one of the main programming languages you use").
 * How regularly do you use Rust?
     * Previously this was phrased in terms of time, but we might want to ask in relation to other languages the programmer uses: e.g., "more than any other language", "one of the top languages", etc.
+* Do users use any of the following specific kinds of coding? How often?
+    - Unsafe Rust
+    - Async Rust
+    - ...
+* What kind of development do you do?
+    - OSs
+    - Network
+    - Games
+    - User-facing Apps (GUI or console)
+    - Mobile apps
+    - Libraries supporting other users
+    - Compilers
+    - DBMS
+    - Other embedded
+    - ...
     
 ## Rust at school
+
 * Are students using Rust in their courses (if such a thing would make sense).
 
 ## Rust at work
 
 We'll want to ensure we're only asking those for whom using a programming language at work even makes sense (i.e., they work in software).
 
 * What percentage of people are using Rust at work?
-* If not using Rust a work, what is preventing it's adoption? 
+* If not using Rust a work, what is preventing it's adoption? Or if it is used, preventing it being used in more places?
```

Comment:
I think there should be a different set of questions for people who use Rust at work.